client/js/_admin/components/lti_installs/accounts.jsx

```diff
@@ -0,0 +1,39 @@
+import React               from 'react';
+import _                   from 'lodash';
+import SubAccount          from './sub_account';
+
+export default class Accounts extends React.Component {
+
+  accounts() {
+    return _(this.props.accounts)
+      .filter({ parent_account_id: null })
+      .sortBy('name')
+      .map(account => (
+        <SubAccount
+          key={`account_${account.id}`}
+          account={account}
+          setAccountActive={this.props.setAccountActive}
+          accounts={this.props.accounts}
+          currentAccount={this.props.currentAccount}
+          isActive={this.props.currentAccount && account.id === this.props.currentAccount.id}
+        />
+      ))
+      .value();
+  }
+
+  render() {
+    return (
+      <ul className="c-filter-list">
+        {this.accounts()}
+      </ul>
+    );
+  }
+}
+
+Accounts.propTypes = {
+  accounts         : React.PropTypes.shape({}).isRequired,
```

Comment:
Can we please not align by the colon? If you want to align the React stuff, whatever, but don't align using the colon. Use the spaces instead.

```javascript
Accounts.propTypes = {
  accounts:         React.PropTypes.shape({}).isRequired,
  setAccountActive: React.PropTypes.func.isRequired,
  ...
}
```